artemis.jh

```diff
@@ -334,7 +346,9 @@ relationship OneToOne {
     ShortAnswerSubmittedText{spot} to ShortAnswerSpot,
     Submission{result} to ExerciseResult{submission},
     LtiUserId{user} to User,
-    ExerciseResult{assessor} to User
+    ExerciseResult{assessor} to User,
+    Feedback{complaint} to Complaint{feedback},
```

Comment:
I think we discussed that complaint is connected to result?

Reply:
As first implementation yes, but we also said that if we have more time we want to split and create for every feedback?

So, since data model changes can be difficult, I was thinking to connect to feedback for future compatibility, and that we allow complain only against the first feedback of a result - if you think is easy to change in a future version, I can link it against Result

Reply:
Please link it to Result, let's keep it simple for now and in the future, we can still change it.

Reply:
Done, I have also added `ComplaintResponse{tutor} to User` since we want to track which tutor is replying to a complaint